Quotient/jobs/mediathumbnailjob.cpp

```diff
@@ -22,7 +22,7 @@ QUrl MediaThumbnailJob::makeRequestUrl(const HomeserverData& hsData, const QStri
                                        std::optional<bool> animated)
 {
     QT_IGNORE_DEPRECATIONS( // For GetContentThumbnailJob
-        return hsData.checkMatrixSpecVersion(u"1.11")
+        return hsData.checkMatrixSpecVersion(u"v1.11")
```

Comment:
Ditto for `DownloadFileJob`, please.

Reply:
DownloadFileJob is correct though, it has v1.11? https://github.com/quotient-im/libQuotient/blob/75d1f2af2a215adeb68663c15d4918ec20e84368/Quotient/jobs/downloadfilejob.cpp#L48

Reply:
Oh, wait... right. That explains why I haven't noticed the problem in my tests.